### ISSUE TITLE
Menu is not read out by NVDA

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51841]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
ScreenReader/NVDA[2018.2],JAWS[18.0.5038],MAS1.3.1

**Repro Steps:**
1.Launch VS Code while NVDA is ON.
2.Navigate to "Menu Bar" at top of the screen after login.
3.Using "Alt+F,Alt+E,Alt+S" these keys navigate to menu bar.
3.Observed that, NVDA is not reading selected menu name of menu bar. 

**Actual:**
While using "Alt+F" key then keyboard focus is on the "File" menu of MenuBar.
but NVDA is not reading it's name as "File".
NVDA is reading it as "Menu SubMenu"

Same issue is observed in JAWS

**Expected:**
NVDA should read the name of selected Menu name.
Here, NVDA should read it like "File Menu" then "Submenu name"
instead of "Menu Submenu"

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
If NVDA is not reading the selected menu name then user will not know which menu is selected and which menu is selected from the menubar.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_MenuBar_ScreenReader.pptx](https://github.com/Microsoft/vscode/files/2101080/Accessibility.A11y_VSCode_MenuBar_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@sbatten this should be fixed with the new custom menu, correct? Assinging to july for discussion

**Comment by sbatten** [comment]
Menu titles are read out now using the custom menubar, but I am not sure if this meets the requirement because this does not happen when the menu is launched via mnemonics. That's because the first menu item is focused and NVDA only reads its aria-label. However, pressing the alt-key focuses the menubar and allows you to navigate the menus and learn the shortcuts. In this way, I believe it meets the accessibility requirements. Awaiting more info.

**Comment by isidorn** [comment]
@sbatten I feel like the menu is still work in progress, thus moving this issue out to August when we can focus on the missing accessibility in the menu.
@AccessibilityTestingTeam-TCS can you please try this out with latest vscode insiders from here https://code.visualstudio.com/insiders/ and check if this is fixed now. Thanks

**Comment by isidorn** [comment]
Let's reopen to give more time to the @AccessibilityTestingTeam-TCS to reply

**Comment by sbatten** [comment]
@AccessibilityTestingTeam-TCS  I don't believe this was tested with the new custom menu, but this one was: https://github.com/Microsoft/vscode/issues/17172 and appears to be the same symptoms. My reply there holds for this issue as well.

**Comment by AccessibilityTestingTeam-TCS** [comment]
**@sbatten** While we press Alt key, NVDA reads name and shortcuts for all menu items and navigates them using arrow keys, seems this fulfills the needed requirements.

**Comment by sbatten** [comment]
Awesome, thanks for the feedback. Closing as resolved.
